docs/summer-of-code.md

```diff
@@ -0,0 +1,55 @@
+# Welcome to Probot's Summer of Code!
+
+In 2018, Probot is participating in Google's Summer of Code and Rails Girls Summer of Code, two amazing programs that help students connect with open source projects.
+
+## FAQ
+
+#### How do I get started with Probot?
+
+We suggest that you build our ["Hello World" app](https://probot.github.io/docs/hello-world/) to get the hang of registering a GitHub App and the base setup of a Probot App.
+
+You can also [join our Slack](https://probot-slackin.herokuapp.com/) and join the **#summer-of-code** channel.
+
+#### Can I claim this cool idea?
+
+We're excited that you're excited, but the Summer of Code ideas in our [ideas repo](https://github.com/probot/ideas) are for the students that are accepted to work on Probot over the summer. Claims to those ideas will be ignored until applicants are officially accepted.
+
+#### Someone has claimed this idea, can I still have it in my application?
+
+Absolutely! See the above answer.
+
+#### I have another cool idea for an app!
+
+Amazing! Be sure to include it in your application.
+
+## Mentors
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center" valign="top">
+        <img width="150" height="150" src="https://github.com/bkeepers.png?s=150">
+        <br>
+        <a href="https://github.com/bkeepers">Brandon Keepers</a>
+      </td>
+      <td align="center" valign="top">
+        <img width="150" height="150" src="https://github.com/gr2m.png?s=150">
+        <br>
+        <a href="https://github.com/gr2m">Gregor Martinez</a>
+      </td>
+      <td align="center" width="20%" valign="top">
+        <img width="150" height="150" src="https://github.com/JasonEtco.png?s=150">
+        <br>
+        <a href="https://github.com/JasonEtco">Jason Etcovitch</a>
+      </td>
+      <td align="center" valign="top">
+        <img width="150" height="150" src="https://github.com/tcbyrd.png?s=150">
+        <br>
+        <a href="https://github.com/tcbyrd">Tommy Byrd</a>
+		<br>
+      </td>
+     </tr>
+  </tbody>
+</table>
+
+## Application Requirements
```

Comment:
I think for now just creating the hello world app and adding a link to a comment created by the app?